app/graphql/types/auth/user.rb

```diff
@@ -10,4 +10,8 @@ class Types::Auth::User < GraphQL::Schema::Object
   field :email, String, null: false do
     description 'Email address of the user'
   end
+
+  def email
+    email.downcase
```

Comment:
The email should be saved with the formatted value so the graphql type will always be downcase. The update can be made here: https://github.com/VetReseau/graphql-auth/blob/eedfa5c190bdf0b3a0f4604c772f38b6867ae36d/app/graphql/mutations/auth/sign_up.rb#L24 and everywhere a user is created into an app that uses this gem

Reply:
@ferland182 after testing it, I realized that the email field is already "downcased". There's nothing to fix actually. I removed this code